### PR TITLE
Feat/rpt

### PR DESCRIPTION
**Pull request by YanLabCHU**

### CODE REVIEW

File: client/src/sagas/patient.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ const getFamily = async (patientDataResponse, mainPatientId) => {
     .filter((member) => !member.entity.reference.includes(mainPatientId))
     .map((member) => member.entity.reference.split('/')[1]);
 
+  if (otherFamilyMemberIds.length === 0) {
```
Comment (francisprovost) [inline client/src/sagas/patient.js:35]: Thanks! 😂

**Review by francisprovost — APPROVED** [review]
👍

**Review by francisprovost — CHANGES_REQUESTED** [review]
Les tests fails malheureusement

**Review by claudia1296 — APPROVED** [review]

**Comment by YanLabCHU** [comment]
Duplicate https://github.com/Ferlab-Ste-Justine/clin-frontend/pull/679

**Pull request closed without merging**